.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
-  - 2.1.0
+  - 2.4.0
+  - 2.3.3
+  - 2.2.6
+  - 2.1.10
```

Comment:
These changes are not related to the nokogiri depedency changes and should go into a separate PR.